src/pages/Details.vue

```diff
@@ -37,55 +42,67 @@
               </div>
               <div class="contentRight">
                 <ul>
-                  Rachel
+                  {{caseDetails.poc}}
                 </ul>
 
                 <ul>
-                  21/08/2020
+                  {{caseDetails.appliedOn}}
                 </ul>
 
                 <ul>
-                  S$1
+                  {{caseDetails.amountRequested}}
                 </ul>
 
                 <ul>
                   Yes
                 </ul>
                 <ul>
-                  21/08/2021
+                  {{caseDetails.lastUpdated}}
                 </ul>
               </div>
             </div>
           </div>
           <div class="border">
             <div class="sectionHeading">REFEREE</div>
-            <div class="sectionBody">
-              <div class="contentLeft">
-                <ul>
-                  Name
-                </ul>
-                <ul>
-                  Occupation
-                </ul>
-                <ul>
-                  Phone
-                </ul>
-                <ul>
-                  Email
-                </ul>
+              <div class="sectionBody">
+                <div class="contentLeft">
+                  <ul>
+                    Name
+                  </ul>
+                  <ul>
+                    Occupation
+                  </ul>
+                  <ul>
+                    Phone
+                  </ul>
+                  <ul>
+                    Email
+                  </ul>
+                </div>
+                <div class="contentRight">
+                  <ul>
+                    {{caseDetails.refereeName}} <!--referee-->
+                  </ul>
+                  <ul>
+                    {{caseDetails.refereeOccupation}}
+                  </ul>
+                  <ul>
+                    {{caseDetails.refereePhone}}
+                  </ul>
+                  <ul>
+                    {{caseDetails.refereeEmail}}
+                  </ul>
+                </div>
               </div>
-              <div class="contentRight"></div>
-            </div>
           </div>
           <div class="border">
             <div class="sectionHeading">Referee Status</div>
             <div class="sectionHeading">
               <div class="select">
                 <select>
                   <option>Unverified</option>
-                  <option>Option 1</option>
-                  <option>Option 2</option>
-                  <option>Option 3</option>
+                  <option>Pending</option>
+                  <option>Verified</option>
                 </select>
               </div>
             </div>
```

Comment:
Amount granted and reference status do  not reflect value from DB
<img width="1209" alt="Screenshot 2021-03-13 at 6 45 23 PM" src="https://user-images.githubusercontent.com/28174698/111027684-c5fc3600-842c-11eb-975d-d458e3482842.png">

Reply:
have updated to show status and amount granted from db